rmgpy/data/thermo.py

```diff
@@ -1470,6 +1472,19 @@ def get_thermo_data_for_surface_species(self, species):
                 else:
                     raise NotImplementedError("Can't remove surface bond of type {}".format(bond.order))
             dummy_molecule.remove_atom(site)
+
+        if len(adsorbed_atoms) == 2:
+            # Bidentate adsorption.
+            # Try to turn adjacent biradical into a bond.
+            try:
+                bond = adsorbed_atoms[0].bonds[adsorbed_atoms[1]]
+            except IndexError:
```

Comment:
Since `adsorbed_atoms[0].bonds` is a dict, I think this should be `KeyError` instead of `IndexError` (@kblondal discovered this when running RMG with this branch)

Reply:
Thanks. making a fixup now